build.gradle

```diff
@@ -169,6 +169,7 @@ project('client') {
         compile project(':contract')
         compile group: 'org.glassfish.jersey.ext', name: 'jersey-proxy-client', version: jerseyVersion, withoutLogger
         compile group: 'org.glassfish.jersey.core', name: 'jersey-client', version: jerseyVersion, withoutLogger
+        compile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
```

Comment:
Do we require this for the client? It should depend on the user application's logging configuration..
This is required only for the `server `subproject right?

Reply:
Removed it from other subprojects.